packages/server/src/jsRunner/tests/jsRunner.spec.ts

```diff
@@ -70,4 +72,278 @@ describe("jsRunner (using isolated-vm)", () => {
       })
     })
   })
+
+  // the test cases here were extracted from templates/real world examples of JS in Budibase
+  describe("should real world tests of JS", () => {
+    const context = {
+      "Unit Value": 2,
+      Quantity: 1,
+    }
+    it("handle test case 1", async () => {
+      const result = await processJS(
+        `
+        var Gross = $("[Unit Value]") * $("[Quantity]")
+        return Gross.toFixed(2)`,
+        context
+      )
+      expect(result).toBeDefined()
+      expect(result).toBe("2.00")
+    })
+
+    it("handle test case 2", async () => {
+      const context = {
+        "Purchase Date": DATE,
+      }
+      const result = await processJS(
+        `
+        var purchase = new Date($("[Purchase Date]"));
```

Comment:
All these are real world js?

Reply:
Yeap - 100% real, I've changed some variable names, if they weren't template JS but apart from that they haven't been altered.